doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -804,11 +804,15 @@ L      Last rule, do not continue.
 QSA    Append the results of the rule to the query string.
 ====== ========================================================================
 
-Variable Expansion
-------------------
+Values and Variable Expansion
+-----------------------------
+You can concatenate values using strings, condition values and variable expansions via the ``+`` operator. The elements in the concatenation must be enclosed in double quotes ``"``::
+
+    add-header CustomHeader "Hello from " + "%{IP:SERVER}" + ":" + "%<INBOUND:LOCAL-PORT>"
```

Comment:
Can you change the documentation to not have the quotes for the condition expansion (e.g. %{IP:SERVER} vs "%{IP:SERVER}")?  I am assuming the quotes are not required and are optional.

Reply:
Not for %{tag} style tags. They're required for %<> style tags. 

I'll push an update for the docs.

Reply:
Sounds like %<> are going away for 9.0.0 - so I am OK with it